### DISCUSSION TITLE
Pipeline technical meetings of September 28 and 29

### DISCUSSION THREAD

**Discussion by bertfrees** [daisy/pipeline#720] [Meetings]
The next set of Pipeline technical meetings are scheduled on September 28 and 29, 2023. The main objective is to finalize the work plan for 2024. Attached is the latest [draft of the work plan](https://github.com/daisy/pipeline/files/12741814/Work.Plan.2024.-.DAISY.Pipeline.draft.docx) for your review.

You can attend any of the meetings depending on your time zone. The timings are:

- Sept 28, at 16 UTC: This time is friendly for America and Europe.
- Sept 29, at 8 UTC: This time is friendly for Europe, Asia and Australia.

**Comment by bertfrees** [comment]
This is the [zoom link](https://www.google.com/url?q=https://us02web.zoom.us/j/87530347979?pwd%3DbmpFQWJrRTNTR1VvYjRwTTlRQ3NCZz09&sa=D&source=calendar&usd=2&usg=AOvVaw0R2EWD25J7M4u-2PNPkPIL).

**Comment by josteinaj** [comment]
I won't be able to attend today. I see that the things I'm most interested in have been entered under "October 2024 - Objective d. Usability improvements and maintenance", which is great :+1:

**Comment by bertfrees** [comment]
This is the [zoom link for Friday 29 September](https://www.google.com/url?q=https://us02web.zoom.us/j/88612661428?pwd%3DRmNlbkN4WWg0R3dRUmdrRnNxdDFUQT09&sa=D&source=calendar&usd=2&usg=AOvVaw3nIAoCUsppbRiC6SM6L0gJ).
